relentless/potential/potential.py

```diff
@@ -40,29 +41,33 @@ class CoefficientMatrix(PairMatrix):
 
     Set coefficient matrix values::
 
-        m['A','A']['energy'] = 2.0
-        m['A,'B']['mass'] = 1.5
-        m['A','B']['energy'] = 3.0
+        m['A','A']['energy'] = 2.0               #access param directly
+        m['A','B'] = {'energy':0.0, 'mass':1.5}  #reset default in full
+        m['A','A'] = {'mass':0.5}                #reset default partially
+        for p in self.params:                    #iteratively acesss params
```

Comment:
Each of these is a different example (with the description given in the comment), so please separate them out.